classes/migration/upgrade/v3_4_0/I7725_DecisionConstantsUpdate.php

```diff
@@ -0,0 +1,268 @@
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 22, 
+            'updated_value' => 20
+        ],
+
+        // \PKP\decision\Decision::RESUBMIT_INTERNAL
+        [
+            'stage_id'      => [], 
+            'current_value' => 23, 
+            'updated_value' => 21,
+        ],
+
+        // \PKP\decision\Decision::DECLINE_INTERNAL
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 24, 
+            'updated_value' => 22,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_ACCEPT_INTERNAL
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 25, 
+            'updated_value' => 23,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_PENDING_REVISIONS_INTERNAL
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 26, 
+            'updated_value' => 24,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_RESUBMIT_INTERNAL
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 27, 
+            'updated_value' => 25,
+        ],
+
+        // \PKP\decision\Decision::RECOMMEND_DECLINE_INTERNAL
+        [
+            'stage_id'      => [], 
+            'current_value' => 28, 
+            'updated_value' => 26,
+        ],
+
+        // \PKP\decision\Decision::REVERT_INTERNAL_DECLINE
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 29, 
+            'updated_value' => 27,
+        ],
+
+        // \PKP\decision\Decision::NEW_INTERNAL_ROUND
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 30, 
+            'updated_value' => 28,
+        ],
+
+        // \PKP\decision\Decision::BACK_FROM_PRODUCTION
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_EDITING], 
+            'current_value' => 31, 
+            'updated_value' => 29,
+        ],
+
+        // \PKP\decision\Decision::BACK_FROM_COPYEDITING
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_SUBMISSION, WORKFLOW_STAGE_ID_INTERNAL_REVIEW, WORKFLOW_STAGE_ID_EXTERNAL_REVIEW], 
+            'current_value' => 32, 
+            'updated_value' => 30,
+        ],
+
+        // \PKP\decision\Decision::CANCEL_REVIEW_ROUND
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_SUBMISSION, WORKFLOW_STAGE_ID_INTERNAL_REVIEW, WORKFLOW_STAGE_ID_EXTERNAL_REVIEW],  
+            'current_value' => 33, 
+            'updated_value' => 31,
+        ],
+
+        // \PKP\decision\Decision::CANCEL_INTERNAL_REVIEW_ROUND
+        [
+            'stage_id'      => [WORKFLOW_STAGE_ID_SUBMISSION, WORKFLOW_STAGE_ID_INTERNAL_REVIEW], 
+            'current_value' => 34, 
+            'updated_value' => 32,
+        ],
+    ];
```

Comment:
Up to you, but these decisions should not exist before 3.4.

Reply:
reason at https://github.com/pkp/ojs/pull/3520#discussion_r958077563 .